test/obb_unit_test.js

```diff
@@ -20,3 +25,53 @@ describe('OBB', function () {
         assert.equal(obb.isSphereAboveXYBox(sphere), true);
     });
 });
+
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+proj4.defs('EPSG:3946', '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+function assertVerticesAreInOBB(builder, extent) {
+    const params = {
+        extent,
+        disableSkirt: true,
+    };
+
+    const geom = new TileGeometry(params, builder);
+    const inverse = new THREE.Matrix4().getInverse(geom.OBB.matrix);
+
+    let failing = 0;
+    const vec = new THREE.Vector3();
+    for (let i = 0; i < geom.attributes.position.count; i++) {
+        vec.fromArray(geom.attributes.position.array, 3 * i);
+
+        vec.applyMatrix4(inverse);
+        if (!geom.OBB.box3D.containsPoint(vec)) {
+            failing++;
+        }
+    }
+    assert.equal(geom.attributes.position.count - failing, geom.attributes.position.count, 'All points should be inside OBB');
```

Comment:
why not write ?
```js
assert.equal(failing, 0, 'All points should be inside OBB');
```

Reply:
With my version, if the test is failing you 2 info: the number of failing vertices and the total number of vertices.